hsctl

```diff
@@ -132,8 +132,6 @@ preflight_hs() {
         # generate nginx configuration file
         if [ "${USE_SSL,,}" = true ]; then
             echo "*** Using SSL: USE_SSL = ${USE_SSL} ***"
-            # use https version of recaptcha
-            sed -i 's!'${HTTP_RECAPTCHA}'!'${HTTPS_RECAPTCHA}'!g' ${HS_PATH}/theme/templates/accounts/_signup_form.html
```

Comment:
I think we should get rid of the sed line and just use a protocol relative URL in _signup_form.html